src/zope/testing/test_renormalizing.py

```diff
@@ -1,35 +1,41 @@
 import unittest
-
+import textwrap
 from zope.testing.renormalizing import strip_dottedname_from_traceback
 
 
 class Exception2To3(unittest.TestCase):
 
     def test_strip_dottedname(self):
         string = """\
-Traceback (most recent call last):
-foo.bar.FooBarError: requires at least one argument."""
+        Traceback (most recent call last):
+        foo.bar.FooBarError: requires at least one argument."""
+        string = textwrap.dedent(string)
         expected = """\
-Traceback (most recent call last):
-FooBarError: requires at least one argument."""
+        Traceback (most recent call last):
+        FooBarError: requires at least one argument."""
+        expected = textwrap.dedent(expected)
         self.assertEqual(expected, strip_dottedname_from_traceback(string))
```

Comment:
This works, but what I had in mind was

```python
        def test_strip_dottedname(self):
            string = textwrap.dedent("""\
                Traceback (most recent call last):
                  ...
                foo.bar.FooBarError: requires at least one argument""")
            expected = textwrap.dedent("""\
                Traceback (most recent call last):
                  ...
                FooBarError: requires at least one argument""")
            self.assertEqual(strip_dottedname_from_traceback(string), expected)
```

Anyway, a minor matter, and while it's still a bit hard to see where a statement begins and ends, the most important thing is that now it's easy to see where a test method begins and ends.  Good enough.